packages/gatsby-cli/src/structured-errors/error-map.ts

```diff
@@ -549,18 +549,12 @@ const errors = {
     docsUrl: `https://www.gatsbyjs.com/docs/reference/gatsby-cli#new`,
   },
   "11614": {
-    text: ({
-      path,
-      filePath,
-      line,
-      column,
-    }): string => `The path "${path}" errored during SSR.
-
-    Edit its component ${filePath}${
-      line ? `:${line}:${column}` : ``
-    } to resolve the error.`,
+    text: (context): string =>
+      stripIndent(`The path "${context.path}" errored during SSR.
```

Comment:
Since it's a `WARNING` the `docsUrl` is unused. Also added `stripIndent`

Reply:
`stripIndent` won't do anything here because it looks for shortest indentation and removes that from every line - in here shortest indentation is empty string (because first line doesn't have any indentation)

Reply:
above comment addressed in https://github.com/gatsbyjs/gatsby/pull/31061/commits/9eb5a5b39532391721b023c7ee374cbdeed59cee